whelk-core/src/main/groovy/whelk/history/Ownership.java

```diff
@@ -0,0 +1,76 @@
+package whelk.history;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+
+public class Ownership {
+    public String m_lastManualEditor;
+    public Instant m_lastManualEditTime;
+
+    public String m_lastEditor;
+    public Instant m_lastEditTime;
+
+    public Ownership(DocumentVersion version, Ownership previousOwnership) {
```

Comment:
Libris-specific stuff, but I think that is ok for now. Let's just note increased "Libris debt" in whelk-core.
(We could later e.g. move interpretation of the strings xl/Voyager/vcopy/APIX etc. to importers?)